src/utils/ChildHair.js

```diff
@@ -2,7 +2,7 @@ import HairStrand from './Hair';
 import Vector from './Vector';
 
 export default class ChildHair extends HairStrand {
-  constructor(parents = Array(3), { drawFunction = () => {}, res = 8 }) {
+  constructor(parents = Array(3), { drawFunction = () => {}, res = 8, bez_res = 4, s_type = 1}) {
```

Comment:
Should the `bez_res` should be the same as the parents? And maybe same for the `res`? And `s_type`?

I'm guessing the interpolation will break if the child hair has a different number of vertices than the parent, and the spread type shouldn't matter since no calculations are actually happening with them other than location which is set by the parents.

Reply:
It doesn't actually matter what they are for the child hairs because the value is never used. outside of the parent constructor. There are a few things like this that ChildHair inherits that aren't useful, maybe it would be better if it wasn't a subclass?

Reply:
Just for clarification, the child hair shouldn't break because it's just forced to interpolate the number of parent final vertices, so the values aren't used I believe

Reply:
I know this won't break it, but it's bad practice to leave unused code in 😃